include/measurement_kit/common/version.hpp

```diff
@@ -6,6 +6,6 @@
 #define MEASUREMENT_KIT_COMMON_VERSION_HPP
 
 // Note: we use semantic versioning (see: http://semver.org/)
-#define MEASUREMENT_KIT_VERSION "0.2.0-alpha.1"
+#define MEASUREMENT_KIT_VERSION "0.2.0-alpha"
```

Comment:
You are right, it was in the commit I reverted. I will add a new commit fixing it!